File: guanabara-java/src/com/danillo/algoritmos/lista02/Exercicio20.java
package com.danillo.algoritmos.lista02;

import java.util.Scanner;

public class Exercicio20 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira um número inteiro: ");
        int x = entrada.nextInt();

        if(x % 2 == 0){
            System.out.println("O número " +x+ " é par.");
        } else {
            System.out.println("O número " +x+ " é ímpar.");
        }
    }
}

File: guanabara-c/lista02/exercicio20.c
#include <stdio.h>

int main(){
    int x;

    printf("Insira um número inteiro: ");
    scanf("%d", &x);

    if(x % 2 == 0){
        printf("O número %d é par.", x);
    }
    else{
        printf("O número %d é ímpar.", x);
    }

    return 0;
}
